### PR TITLE
chore: update ci workflows and deps

### PR DESCRIPTION
**Pull request by SrikarMannepalli**

### CODE REVIEW

**Comment by SrikarMannepalli** [comment]
Waiting for https://github.com/hypertrace/event-store/pull/7

File: .github/workflows/pr-build.yml [hunk 1]
```diff
@@ -9,50 +9,39 @@ on:
...
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
         with:
-          args: build dockerBuildImages
+          args: assemble dockerBuildImages
+
+      - name: Run Trivy vulnerability scanner
+        uses: hypertrace/github-actions/trivy-image-scan@main
+        with:
+          image: hypertrace/attribute-service
```
Comment (aaron-steinfeld) [inline .github/workflows/pr-build.yml:36]: wrong image

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-hypertrace-framework = "0.1.48"
-lombok = "1.18.22"
-jackson = "2.14.1"
-kafka = "6.0.1-ccs"
-kotlin = "1.4.32"
+hypertrace-grpcutils = "0.12.0"
+hypertrace-framework = "0.1.52"
+lombok = "1.18.28"
+jackson = "2.15.2"
+kafka = "7.2.1-ccs"
```
Comment (aaron-steinfeld) [inline gradle/libs.versions.toml:9]: try to switch over to use the kafka bom

Comment (SrikarMannepalli) [reply gradle/libs.versions.toml]: Done.

**Comment by SrikarMannepalli** [comment]
> lgtm code wise, looks like some vulns remaining though

`netty-handler-4.1.87.Final.jar (pkg:maven/io.netty/netty-handler@4.1.87.Final, cpe:2.3:a:netty:netty:4.1.87:*:*:*:*:*:*:*) : CVE-2023-34462` is coming up after the latest commit. Doesn't come up on local though.

**Comment by aaron-steinfeld** [comment]
Brand new, bad luck. Let me go update upstream.

**Comment by aaron-steinfeld** [comment]
https://github.com/hypertrace/java-grpc-utils/pull/47

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -2,11 +2,11 @@
 protoc = "3.21.12"
 grpc = "1.56.0"
 gson = "2.9.0"
-hypertrace-grpcutils = "0.12.0"
+hypertrace-grpcutils = "0.12.1"
 hypertrace-framework = "0.1.52"
+kafka-streams-framework = "0.2.10"
```
Comment (aaron-steinfeld) [inline gradle/libs.versions.toml:7]:
```suggestion
kafka-streams-framework = "0.2.11"
```

**Review by aaron-steinfeld — COMMENTED** [review]
We should consolidate the workflows and add scheduling too, but can happen in separate PR. Example: https://github.com/hypertrace/attribute-service/pull/170

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**